src/Pyz/Yves/ConfPage/Controller/IndexController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of the Spryker Commerce OS.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Pyz\Yves\ConfPage\Controller;
+
+use Spryker\Yves\Kernel\View\View;
+use SprykerShop\Yves\ShopApplication\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+
+class IndexController extends AbstractController
```

Comment:
This controller will have an action where the configuration will need to be collected: availability and price calculated to be returned.